participants/AiData.md

```diff
@@ -0,0 +1,68 @@
+# <Ai Data>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+- AI Data
+- Allen
+
+## (1) Brief Project Description
+
+**AI data focuses on AI data services, providing safe and reliable data for global AI enterprises.**
+**Provide customers with all types of ARTIFICIAL intelligence data covering voice, image, text, etc.**
+**Ai Data is committed to becoming a reliable and secure platform for ARTIFICIAL intelligence data.**
+## (2) Link to Project UI
+
+- [ui](https://github.com/a8159236/AiData/tree/master/images)
+- [source](https://github.com/a8159236/AiData)
+
+
+## (3) What does your application/UI do?
+
+AI data is a professional artificial intelligence data providing platform, which interacts with filecoin network, and stores AI data by filecoin network.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+**WAV**
```

Comment:
You will need to specify the particular datasets used and submit a PR for that data in the datasets.md file.   Provisionally approving.